src/goose/_logger.py

```diff
@@ -14,6 +92,19 @@ def setup_logging(log_file_directory: Path, log_level: str = "INFO") -> None:
     formatter = logging.Formatter("%(asctime)s - %(name)s - %(levelname)s - %(message)s")
     file_handler.setFormatter(formatter)
 
+    trace_logger = logging.getLogger(_TRACE_LOGGER_NAME)
+    trace_logger.setLevel(log_level)
+    trace_handler = TimedRotatingFileHandler(
+        log_file_directory / _TRACE_LOGGER_FILE_NAME, when="midnight", interval=1, backupCount=7
```

Comment:
random style thing that you can ignore this as well - but if you add a trailing comma at the end of `backupCount=7` you can force `ruff` to put all of the parameters on a newline so it's easier to do pairwise diffs